changelog/unreleased/kong/fix-wasm-module-branch.yml

```diff
@@ -0,0 +1,3 @@
+message: use NGX_WASM_MODULE_BRANCH environment variable to set ngx_wasm_module repository branch when building Kong.
+type: bugfix
+scope: Core
```

Comment:
"chore"?

Reply:
I agree it would be a better option, but it is not one of the [types available in the new changelog format](https://github.com/Kong/kong/blob/3.5.0/changelog/changelog-template.yaml#L2).